.gitignore

```diff
@@ -51,3 +51,5 @@ bin
 
 # Vim
 .swp
+
+.gitpod.yml
```

Comment:
just confirming this is intentional (no objection)

Reply:
Yeah figured it doesn't hurt